src/screens/Learn/Glossary/index.ts

```diff
@@ -0,0 +1,3 @@
+import Glossary from './Glossary';
```

Comment:
are individual `index` files overkill? maybe just the one main `index` file in the Learn folder suffices?

Reply:
yeah, since they share most of the styles we might as well skip the index pages